mkdocs-base.yml

```diff
@@ -178,17 +178,20 @@ nav:
           - get-started/roles-and-permissions/use_case.md
       - get-started/query-analytics.md
       - get-started/advisors.md
+      - get-started/HA.md
```

Comment:
@rasika-chivate this entry seems old, as we then moved the HA page under how-to.  Should we remove this line? see how it looks like in the preview, FYI (see that "none" after "working with advisors")

<img width="272" alt="none" src="https://github.com/percona/pmm-doc/assets/31849787/f6112844-59d6-466f-8a6b-6eb64218b905">

Reply:
Done